tests/helpers.py

```diff
@@ -331,9 +335,12 @@ def get_all_storageclass_name():
     return storageclass
 
 
-def delete_all_storageclass():
+def delete_storageclass(sc_name):
```

Comment:
Please verify no else is using this function as you changed its name

Reply:
Checked no one else using this function